lib/data/multipleBackendGateway.js

```diff
@@ -146,6 +146,20 @@ const multipleBackendGateway = {
         }
         return cb();
     },
+
+    abortMPU: (key, uploadId, location, bucket, log, cb) => {
+        const client = clients[location];
+        if (client.clientType === 'aws_s3') {
```

Comment:
I'll do a followup PR to change that logic in the other three MPU functions as well